alvr/client_openxr/src/lib.rs

```diff
@@ -109,6 +109,23 @@ fn default_view() -> xr::View {
     }
 }
 
+// This exists to circumvent dead-code analysis
+fn create_session(
+    xr_instance: &xr::Instance,
+    xr_system: xr::SystemId,
+    graphics_context: &GraphicsContext,
+) -> (
+    xr::Session<xr::OpenGlEs>,
+    xr::FrameWaiter,
+    xr::FrameStream<xr::OpenGlEs>,
+) {
+    unsafe {
+        xr_instance
+            .create_session(xr_system, &graphics::session_create_info(graphics_context))
+            .unwrap()
+    }
+}
```

Comment:
I'm unsure how does this remove the lint

Reply:
I have no clue, it just does.
Presumably some shenanigans about unsafe block as the only thing in a function
I'm mainly happy that it does